mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return trimWord();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
```

Comment:
Hi. This one is similar to the top in exercises, where u got the trim function right.